config.toml

```diff
@@ -57,3 +57,45 @@ localUrl = "/about"
 [[params.links]]
 name = "Get Started"
 localUrl = "/get-started"
+
+[menu]
+
+[[menu.docs]]
+    name = "JavaScript"
+    url = "https://open-telemetry.github.io/opentelemetry-js/"
+
+[[menu.docs]]
+    name = "Go"
+    url = "https://github.com/open-telemetry/opentelemetry-go/blob/master/README.md"
+
+[[ menu.docs ]]
+    name = "Python"
+    url = "https://opentelemetry-python.readthedocs.io/"
```

Comment:
@c24t would you prefer for the OpenTelemetry website to point to the docs on readthedocs.io or on GitHub?